startup.sh

```diff
@@ -87,4 +98,6 @@ case $ARCH in
 		;;
 esac
 
-$JAVA_EXE -Dsun.java2d.dpiaware=false -Djava.library.path=${NATIVE_LIB_PATH} -classpath "$OPENHOSPITAL_CLASSPATH:$CLASSPATH" org.isf.menu.gui.Menu "$@"
+######### OPENHOSPITAL STARTUP
+
+cd $OPENHOSPITAL_HOME&& $JAVA_BIN -Dsun.java2d.dpiaware=false -Djava.library.path=${NATIVE_LIB_PATH} -classpath "$OPENHOSPITAL_CLASSPATH" org.isf.menu.gui.Menu "$@"
```

Comment:
nitpick: I'm not a big fan of scripts that change `$PWD`. I suggest using `pushd $OPENHOSPITAL_HOME` and then `popd` after the execution.

Reply:
Me neither :-) I commented that it is a bad trick, but there is no other way of having the script working if launched outside of the OPEN_HOSPITAL directory.
That's the reason why I asked how we can strip the references to the absolute paths in the code / properties file.
You can  try the behaviour very simply, just comment out the cd $OPENHOSPITAL_HOME and try something like

cd /tmp
/usr/local/OpenHospital/startup.sh

Reply:
yes, absolute paths in code are definitely something we should change.
in the meantime, my point was more about restoring the PWD the user was in, when finishing the execution - you can do it with `pushd` / `popd`.